db/db_impl/db_impl.cc

```diff
@@ -2063,10 +2063,11 @@ Status DBImpl::GetEntity(const ReadOptions& _read_options, const Slice& key,
     return Status::InvalidArgument(
         "Cannot call GetEntity without PinnableAttributeGroups object");
   }
+  Status s = Status::OK();
```

Comment:
very nit: `Status s` defaults to create a OK status, although this explicit assignment may be more readable.